appintro/src/main/java/com/github/appintro/AppIntroFragment.kt

```diff
@@ -5,6 +5,7 @@ import androidx.annotation.ColorRes
 import androidx.annotation.DrawableRes
 import androidx.annotation.FontRes
 import com.github.appintro.model.SliderPage
+import java.util.UUID
```

Comment:
Can you undo this change?

Reply:
Sure, that was accidentially leftover from an older commit.

To the other function:
It is only a tiny wrapper but one that, i imagine, will be useful to most people, since it takes care of the "combining of the permission and slide. Should i really remove it?